src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiImageArchiveInfo.java

```diff
@@ -71,22 +71,31 @@ public void setRootId( Boolean rootId )
         this.rootId = rootId;
     }
 
+    public static KojiArchiveInfo addImageArchiveInfo( KojiImageArchiveInfo imageArchiveInfo, KojiArchiveInfo archiveInfo )
```

Comment:
If these change the internal state of the KojiArchiveInfo instance, why make them static?

Reply:
> If these change the internal state of the KojiArchiveInfo instance, why make them static?

The instance is of type `Koji*ArchiveInfo`, not `KojiArchiveInfo`, and `KojiArchiveInfo` is what it's changing.

I guess these methods could be moved to `KojiArchiveInfo` instead and made non-static, if that's what you were thinking.

However, we have a similar method for `BuildTypeInfo`, which also might be a candidate for changing in that case:

`public static KojiBuildInfo addBuildTypeInfo( KojiBuildTypeInfo buildTypeInfo, KojiBuildInfo buildInfo )`